### DISCUSSION TITLE
ubuntu与manjaro 使用davfs2 、rclone 、webdavfs 挂载都出现无法读写的情况

### DISCUSSION THREAD

**Discussion by XG520** [AlistGo/alist#5559] [Q&A]
ubuntu与manjaro 使用davfs2 、rclone 、webdavfs 挂载都出现无法读写的情况
![1700732827235](https://github.com/alist-org/alist/assets/83686186/7b8ef4bd-82a5-4d4f-a2bb-7e2fba3c2f21)
![image](https://github.com/alist-org/alist/assets/83686186/f7997a85-dcf1-4c48-af55-b1dcbe4b237d)

**Comment by WXZhao7** [comment]
davfs2的话，需要配置`use_locks 0`，rclone测试阿里云/天翼云/OneDrive/123云盘这些都没问题
